examples/capture-eye-html-design/js/main.js

```diff
@@ -0,0 +1,355 @@
+  var existingTooltip = document.getElementById('custom-tooltip');
+
+  if (existingTooltip) {
+    // If tooltip exists, remove it
+    existingTooltip.remove();
+  } else {
+    // If no tooltip, create a new one
+    var newTooltip = document.createElement('div');
+    newTooltip.id = 'custom-tooltip';
+    newTooltip.textContent =
+      'digitalSourceType is a controlled vocabulary that indicates from which source a digital media was created.';
+    // style tooltip
+    newTooltip.style.position = 'absolute';
+    newTooltip.style.width = '220px';
+    newTooltip.style.padding = '8px';
+    newTooltip.style.backgroundColor = 'black';
+    newTooltip.style.color = 'white';
+    newTooltip.style.textAlign = 'center';
+    newTooltip.style.fontFamily = "'Noto Sans', Arial, sans-serif";
+    newTooltip.style.zIndex = 1000;
+    newTooltip.style.borderRadius = '5px';
+
+    // Get the position of the target element
+    var targetElement = event.target;
+    var targetRect = targetElement.getBoundingClientRect();
+
+    // Position the tooltip below the target element
+    newTooltip.style.left = targetRect.left - 150 + 'px';
+    newTooltip.style.top = targetRect.bottom - 130 + 'px';
+
+    // Append the new tooltip to the body
+    document.body.appendChild(newTooltip);
+
+    setTimeout(() => {
+      newTooltip.remove();
+    }, 1300);
+  }
+}
+
+/**
+ * Displays a snackbar notification with the specified text for a short duration.
+ *
+ * @param {string} text - The text to display in the snackbar notification.
+ */
+function showSnackbar(text) {
+  var x = document.getElementById('snackbar');
+  x.className = 'show';
+  x.innerText = text;
+  setTimeout(function () {
+    x.className = x.className.replace('show', '');
+  }, 3000);
+}
+
+/**
+ * Copies the text content of an element with the given ID to the clipboard.
+ * Displays a snackbar notification upon successful copy or if the element is not found.
+ *
+ * @param {string} elementId - The ID of the element whose text content will be copied.
+ */
+function copyToClipboard(elementId) {
+  // Get the element by its ID
+  var element = document.getElementById(elementId);
+
+  if (element) {
+    // Create a temporary textarea element to copy the text to the clipboard
+    var textarea = document.createElement('textarea');
+    textarea.value = element.textContent || element.innerText;
+    document.body.appendChild(textarea);
+    textarea.select();
+    document.execCommand('copy');
+    document.body.removeChild(textarea);
+
+    showSnackbar('NID copied to clipboard');
+  } else {
+    showSnackbar('Element with ID ' + elementId + ' not found.');
+  }
+}
+
+/**
+ * Fetch data from a remote API using the specified NID and user token.
+ * If successful, update the UI with the retrieved data.
+ * If there's an error, update the UI with an error message.
+ * @async
+ */
+async function fetchData() {
+  const nid = 'bafkreibfqutz3xybzgudrwfs43s7dlgmx2tbvqj5r7ivxzplwrahdoyrfi ';
```

Comment:
Is this supposed to be a hardcoded value?

Reply:
@shc261392, it was early implementation and not used anymore. In the beginning I didn't know should it be separate files or 1 big HTML. 

Later I realized we need 1 big HTML therefore I refactored to 1 big file starting from commit.

- [WIP(example-sona): add sona imaginary HTML](https://github.com/numbersprotocol/capture-eye/pull/2/commits/86af3de20204d7ecb17671bd9afe0ca897ba48ec)
- [WIP(example-sona): add placeholder for HTML/CSS/JS](https://github.com/numbersprotocol/capture-eye/pull/2/commits/94156db0fccd11bf623baacc1c4282e4fbc1a531)
- [WIP(example-sona): add CSS](https://github.com/numbersprotocol/capture-eye/pull/2/commits/eb62c5f636adebc5facf0d36d87cb4d566e8c50c) (here I move css from examples/capture-eye-html-design/css to 1 big file sona-page-with-sample-assets.html
- ... some other commits, moving fonts etc ...
- [WIP(example-sona): add JS](https://github.com/numbersprotocol/capture-eye/pull/2/commits/aa0b6c96d9d33e288920101d6bb3d686a13b624d) (here I move js from examples/capture-eye-html-design/js to 1 big file sona-page-with-sample-assets.html

I will add one more commit which will 

- delete examples/capture-eye-html-design/* (it does not match the design)
- and use sona-page-with-sample-assets.html as main example (since it passed QA and matches the design)

After that I will ping you one more time with code review details.